src/main/webapp/scripts/tutorial-scripts/tutorial-template.js

```diff
@@ -0,0 +1,35 @@
+// Template for mustache rendering tutorial pages
+// Template in Javascript because the template in HTML created linter errors.
+
+const tutorialTemplate =
+'<div id="header">\n' +
+'<h1>{{toolName}}</h1>\n' +
+'<h2>{{toolDescription}}</h2>\n' +
+'<a href="{{toolLink}}" target="_blank" rel="noopener"\n' +
+'>{{toolName}} Link</a>\n' +
+'</div>\n' +
+'<div id="content">\n' +
+'<div id="content-left">\n' +
+'<h1>Useful Links</h1>\n' +
+'{{#usefulLinks}}\n' +
+'<h3>{{linkTitle}}</h3>\n'+
+'{{#links}}' +
+'<li>' +
+'<a href="{{url}}" target="_blank" rel="noopener"\n' +
+'>{{linkName}}</a>' +
+'<li>' +
+'{{/links}}' +
+'{{/usefulLinks}}\n' +
+'</div>\n' +
+'<div id="content-right">\n' +
+'<h1>Useful Videos</h1>\n' +
+'{{#usefulVids}}\n' +
+'<h2>{{linkTitle}}</h2>\n' +
+'<iframe class="tutorials-vid" src="{{vidLink}}"\n' +
+'allow="accelerometer; autoplay;\n' +
+'encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>\n' +
+'{{/usefulVids}}\n' +
+'</div>\n' +
+'</div>';
```

Comment:
I think you can use backticks ``` ` ``` for block quotes (see here: https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript), instead of having `\n' +` at the end of each line.

We should also include proper indentation here to keep it readable.

Reply:
Changed to backtiks for multiline strings, removed newline at the end of each line.

Reply:
Oh I meant putting the whole template in a single set of backticks 😄 

```
const tutorialTemplate = `
<div id="header">
  <h1>{{toolName}}</h1>
  ...
</div>
`;

export {tutorialTemplate};
```